content/pages/meetup/2023/03-nycu.rst

```diff
@@ -0,0 +1,44 @@
+========================================
+Meetup 2023 March in NYCU
+========================================
+
+:date: 2023-03-11 15:08
+:url: meetup/2023/03-nycu
+:save_as: meetup/2023/03-nycu.html
+
+Meetup 2023 March in NYCU
+========================================
+
+* Date: March 15th, 22th and 29st, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+To join the meetup, please bring your laptop. You are also very welcome to bring your project. Please `contact us
+<#contact-us>`__ if you have any questions.
+
+Meetup is an event providing space for people to work on open source
+projects together. We welcome any subjects that may interest the attendees,
+and especially encourage code for science, engineering, and technology, which
+demand more critical discussions than other applications of computer
+programming.
+
+We would like to provide a supportive and friendly environment for all attendees to support more developers
+to join in the open-source communities. 
+
+Venue
+-----
+
+`國立陽明交通大學 工程三館 3 樓 329 室 <https://goo.gl/maps/TgDYwohB3CBmQgww9>`__.
```

Comment:
Could you please also add English description for the venue?  It will help non-Chinese users to access.

Reply:
Sure!